crates/nostr/src/event/tag.rs

```diff
@@ -337,6 +341,7 @@ pub enum Tag {
     Preimage(String),
     Relays(Vec<UncheckedUrl>),
     Amount(u64),
+    Lnurl(String),
```

Comment:
Would it make more sense to use Url instead of String here?

Reply:
No, because it should be a bech32 string like `lnurl1dp68gurn8ghj7um5v93kketj9ehx2amn9uh8wetvdskkkmn0wahz7mrww4excup0dajx2mrv92x9xp`.